.github/workflows/test.yml

```diff
@@ -70,6 +70,9 @@ jobs:
           # https://github.com/rails/rails/issues/40938
           - ruby: '3.0'
             gemfile: 'rails_5.2'
+          # rails 7 requires ruby >= 2.7.0
+          - ruby: '2.6'
```

Comment:
```suggestion
          - ruby: '2.7'
```

Reply:
This is an exclusion list, so I make sure we don't run tests for ruby 2.6 + rails 7.0